public/static/docs/command-reference/run.md

```diff
@@ -52,6 +52,14 @@ captures data and <abbr>caches</abbr> relevant <abbr>data artifacts</abbr> along
 the way. See [this example](/doc/get-started/example-pipeline) to learn more and
 try creating a pipeline.
 
+### Running other dvc commands in parallel
```

Comment:
I'm not sure about this ... why do we mention this here and in repro and not in `dvc add`? 

We also def missing any reasonable context about the `project lock`.

It feels that it can be a general `user guide` section that describes locking and when and how do we release it. How to find those files, etc.

Reply:
@shcheklein because `dvc add` doesn't release repo lock, but `repro/run` do release it. So for `add` the section would be `running dvc run/repro in parallel` which is odd to say the least. Eventually we will release repo lock on checksum computations/copies too, so we will alter the docs accordingly.

Reply:
Ok, I guess this behaviour is simply an expected one, hence why so many people were running into it. So it is not worth documenting it here. Will move to user-guide.

Reply:
Ok, removed that note entirely. Will wait until we do similar things on hash computation/copying/downloading and will then create a `locking` article, that would be too incomplete to bother with right now.